webknossos-datastore/app/com/scalableminds/webknossos/datastore/dataformats/BucketProvider.scala

```diff
@@ -17,7 +17,7 @@ import scala.concurrent.{Await, ExecutionContext, Future}
 
 trait BucketProvider extends FoxImplicits with LazyLogging {
 
-  def loadFromUnderlying(readInstruction: DataReadInstruction)(implicit ec: ExecutionContext): Fox[Cube] = Fox.empty
+  def loadFromUnderlying(readInstruction: DataReadInstruction)(implicit ec: ExecutionContext): Fox[Cube] = Fox.empty(ec)
```

Comment:
@fm3 Removing `Implicits.global` does not work for `BucketProvider.scala`, as it seems to be needed elsewhere.